ck3-tiger.conf

```diff
@@ -0,0 +1,19 @@
+filter = {
+    trigger = {
+        NOT = {
+            OR = {
+                AND = {
+                    key = missing-localization
+                    OR = {
+                        text = "spanish"
+                        text = "french"
+                        text = "german"
+                        text = "russian"
+                        text = "polish"
+                    }
```

Comment:
This section seems odd. Why are we only ignoring these localizations? There are many, many more potential localizations. Does Tiger only monitor these ones?

Reply:
These are the currently included languages in the mod, and it's also easy for someone who want's to complete localization to only remove the relevant language. The reason why i don't just block the entire message is to warn about English localization missing.

Reply:
Is there a problem with warnings for missing localization? Warnings are just warnings. So long as it isn't throwing errors, that should be fine (and even useful for seeing gaps in our localizations).

Reply:
I see it as a problem as it buries the errors and warnings useful to most people  under dozens of missing localization messages. These localization messages are only useful to the respective people who create localization for their language, so i don't think it'd be wise to include them in a default config.
I think it'd be better to hide them by default, but maybe there should be an entry in the readme to show translators how to comment out their language.